### PR TITLE
Convert Css to an opaque type

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

File: common/src/main/scala/react/common/style/package.scala [hunk 1]
```diff
@@ -1,162 +1,158 @@
...
+opaque type Css = List[String]
 
-    given Eq[Style]     = Eq.by(_.styles)
-    given Monoid[Style] = new Monoid[Style] {
-      override val empty: Style                       = Style(Map.empty)
-      override def combine(a: Style, b: Style): Style =
-        Style(a.styles ++ b.styles)
-    }
+object Css:
+  final class ClassnameCssOps(cn: (js.UndefOr[String], js.UndefOr[Css])) {
```
Comment (cquiroz) [inline common/src/main/scala/react/common/style/package.scala:116]: Trying to convert this to a an extension method proved to be a can of worms, with `toJs` colliding with other methods.
I may revisit this once we get rid of sui

Comment (toddburnside) [reply common/src/main/scala/react/common/style/package.scala:116]: When we get rid of SUI, we may no longer need this.

Comment (cquiroz) [reply common/src/main/scala/react/common/style/package.scala:110]: It is used in a couple of other facades
Maybe I could add an extension with another name (method different than `toJs`) and port only those (except SUI)

Comment (cquiroz) [reply common/src/main/scala/react/common/style/package.scala:110]: Actually it seems to work right away, amazingly metals refactoring did handle a rename

Comment (toddburnside) [reply common/src/main/scala/react/common/style/package.scala]: It appears it did work. Of course, not I can't find your actual changes in the 116 files. 👅

**Pull request merged**